CHANGELOG.md

```diff
@@ -1,5 +1,43 @@
 # Change Log
 
+## [v1.1.4](https://github.com/ably/ably-ruby/tree/v1.1.4)
+
+[Full Changelog](https://github.com/ably/ably-ruby/compare/v1.1.4.rc...HEAD)
+
+**Implemented enhancements:**
+
+- statesman dependency very outdated [\#199](https://github.com/ably/ably-ruby/issues/199)
+- Add support for custom transportParams [\#176](https://github.com/ably/ably-ruby/issues/176)
+- Re-enable imempotency tests as part of 1.1 release [\#174](https://github.com/ably/ably-ruby/issues/174)
+- Ensure request method accepts UPDATE, PATCH & DELETE verbs [\#168](https://github.com/ably/ably-ruby/issues/168)
+- my-members presenceMap requirement change for 1.1 [\#163](https://github.com/ably/ably-ruby/issues/163)
+- Add ChannelProperties as part of 1.0 spec \(RTL15\) [\#112](https://github.com/ably/ably-ruby/issues/112)
+
+**Fixed bugs:**
+
+- client\_id should be passed as clientId [\#159](https://github.com/ably/ably-ruby/issues/159)
+
+**Closed issues:**
```

Comment:
I would consider dropping this header and just putting them all under "Fixed bugs"... the changelog generator gets them from labels and those aren't always appropriate.

Reply:
done - 810730b